### PR TITLE
Update take your account with you success messages

### PR DESCRIPTION
**Pull request by ewjordan**
Without email:
screen shot 2017-08-22 at 5 11 06 pm

With email:
screen shot 2017-08-22 at 5 12 06 pm

### CODE REVIEW

**Comment by ewjordan** [comment]
@poorvasingal for text verification

File: dashboard/config/locales/en.yml [hunk 1]
```diff
@@ -260,8 +260,8 @@ en:
     registrations:
       user:
         user_type_change_email_mismatch: "The email address you provided doesn't match the email address for this account"
-        personal_login_created_email: 'You created a personal login successfully. You can now log into Code.org with your email and password'
-        personal_login_created_username: 'You created a personal login successfully. You can now log into Code.org with your username and password'
+        personal_login_created_email: 'You created a personal login successfully. You can now sign into your account with your email and password'
```
Comment (ashercodeorg) [inline dashboard/config/locales/en.yml:263]: Presumably we cannot show the email (like we do with the username) since we may not know it?

Comment (ewjordan) [reply dashboard/config/locales/en.yml:263]: Right, these are student accounts and we don't want to save it

**Comment by poorvasingal** [comment]
Looks good to me! Thanks!

**Pull request merged**